### PR TITLE
Cleaner markdown visualisation

### PR DESCRIPTION
**Pull request by Octoton**
I am a fan of the scripted .md file but the user experiences is a little bit glaring in my opinion.

I this PR I am adding inertial scrolling, removing all the header line but keeping some for licenses:
licenses
changelog

### CODE REVIEW

**Comment by Octoton** [comment]
@soncaokim are you ok with this markdown representation ? (for changelog and licenses)

File: app/src/main/res/layout/dialog_markdown_view.xml [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+    android:layout_height="match_parent">
 
     <TextView
         android:id="@+id/markdown_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/default_item_margin"
-        android:textIsSelectable="false"
-        android:scrollbars="vertical" />
+        android:textIsSelectable="true"
```
Comment (soncaokim) [inline app/src/main/res/layout/dialog_markdown_view.xml:11]: If I'm not wrong I need to disable the `textIsSelectable` attribute to be able to open URL links.

Comment (soncaokim) [reply app/src/main/res/layout/dialog_markdown_view.xml:11]: Confirmed that the links are no longer clickable.

Comment (Octoton) [reply app/src/main/res/layout/dialog_markdown_view.xml]: Done

**Comment by soncaokim** [comment]
Good for me. Can you please fix the conflict though?

**Review by soncaokim — APPROVED** [review]

**Pull request merged**